data/map planets.txt

```diff
@@ -4958,3 +4958,67 @@ planet Zydee
 	landscape land/enceladus_1
 	description `Although this moon initially seems barren and lifeless, there is evidence that the nearby Gegno Scin have recently attempted to set up an outpost here. Several partially-constructed buildings and landing zones are scattered about the surface, but none of them show any signs of activity, likely due to recent warfare with the Gegno Vi. Despite this, a small number of satellites actively orbit Zydee, likely providing important tactical information such as the volume of system traffic.`
 	government Uninhabited
+
+
+# Pirate Stuff
+
+planet Gagarin
+	attributes CCOR factory urban
+	landscape land/city0
+#	description "Gagarin is the oldest-settled planet in the CCOR and the most populated. All the settlements of Gagarin are connected to a planetary transport and infrastructure grid... though the grid hasn't been able to keep pace with demand from a growing population. Although they were designed to be expanded with demand over time, such resource-intensive infrastructure projects became harder and harder to supply the materials for."
+#	spaceport "The heart of the planetary infrastructure network the spaceport is perpetually crowded. Materials are constantly coming and going with ships landing, taking off, or being moved to underground hangars if they can't be unloaded quickly. The spaceport,at least, appears as if, albeit just barely, it's able to meet demand"
```

Comment:
probably better as "The space port, at least, appears as if it's able to meet demand- albeit just barely."